publisher-provider/publisher-provider-app/src/main/java/nl/idgis/publisher/provider/database/messages/DescribeTable.java

```diff
@@ -4,20 +4,38 @@
 
 public class DescribeTable extends Query {
 
-	private static final long serialVersionUID = -7243209795589257031L;
+	private static final long serialVersionUID = 4436618297183961160L;
 	
 	private final String tableName;
+
+	private final String scheme;
 	
 	public DescribeTable(String tableName) {
-		this.tableName = tableName;
+		String[] schemaTableParts = tableName.split("\\.");
+
+		switch (schemaTableParts.length) {
+			case 1: // only table
+				this.scheme = null;
+				this.tableName = schemaTableParts[0];
+				break;
+			case 2: // includes scheme in tablename
+				this.scheme = schemaTableParts[0];
+				this.tableName = schemaTableParts[1];
+				break;
+			default: // includes db and scheme in tablename

```

Comment:
Het db gedeelte wordt hier weggegooid. Hoe zeker weten we dat dit niet tot complicaties gaat leiden?

Reply:
Dit wordt nergens in een query gebruikt en deze info hadden we eerste niet. Volgens mij leidt het daarom niet tot complicaties